### DISCUSSION TITLE
DeprecationWarning: The transformer API will be deprecated in Kedro 0.18.0

### DISCUSSION THREAD

**Discussion by atsangarides** [kedro-org/kedro#1504] [Idea]
full deprecation warning

> DeprecationWarning: The transformer API will be deprecated in Kedro 0.18.0.Please use Dataset Hooks to customise the load and save methods.For more information, please visithttps://kedro.readthedocs.io/en/stable/07_extend_kedro/02_hooks.html

I have a custom dataset that executes an Athena query and reads the result into pandas DataFrame. This class inherits from `AbstractDataSet`. Not quite clear how I can switch to using hooks instead.. I read the docs but there is no clear example for custom datasets, as the instructions still suggest writing a custom class for that... Any help really appreciated

**Comment by datajoely** [comment]
Hello - I think there is a version of kedro where this warning triggers incorrectly.

Since 0.18.x is now out the easiest way to be 100% sure if to create a sample project and test things work correctly.

**Reply by atsangarides** [reply]
amazing!
We are using `0.17.6` and we want to upgrade to 0.18.x to use more recent py versions, and trying to estimate the time it would take to do that.
Any way to then filter logging messages at a global level? Did not manage to do that
